Makefile

```diff
@@ -12,7 +12,7 @@ find-and-split-static: find-and-split-static.cpp
 	$(CXX) -fuse-ld=lld -glldb $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions $< -o ./find-and-split-static
 
 split-llvm-extract: split-llvm-extract.cpp
-	$(CXX) ${CXXFLAGS} -fuse-ld=lld -O0 -g3 $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions $< -o ./split-llvm-extract
+	$(CXX) ${CXXFLAGS} -fuse-ld=lld -g3 $(shell $(LLVM_CONFIG) --link-shared --cppflags --ldflags --system-libs --libs) -std=c++17 -fno-exceptions $< -o ./split-llvm-extract
```

Comment:
Out of interest what does `-g3` do (specifically the `3` part)?

Reply:
There are multiple debug levels. And I wanted to force a specific one, that shows some macro information too.

https://gcc.gnu.org/onlinedocs/gcc/Debugging-Options.html

Reply:
Aha OK!